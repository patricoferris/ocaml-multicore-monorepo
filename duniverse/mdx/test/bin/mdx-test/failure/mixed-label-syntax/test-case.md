You cannot mix legacy labels with HTML comment labels for a single block

<!-- $MDX set-SOME_VAR="a" -->
```sh set-SOME_OTHER_VAR="b"
echo $SOME_VAR
a
echo $SOME_OTHER_VAR
b
```
