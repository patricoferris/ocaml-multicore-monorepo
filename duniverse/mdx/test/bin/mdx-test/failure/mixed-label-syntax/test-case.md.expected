[mdx] Fatal error: File "test-case.md", line 3: invalid code block: cannot mix both block labels syntax
